transfers/views/deadline_views.py

```diff
@@ -0,0 +1,31 @@
+from transfers.constants import TransferType
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.views import generic
+ 
+from transfers.models import DeadlineModel
+ 
+from transfers.utils import post_PSD_post
+from transfers.forms import PSDForm
+
+ 
+class PSDview(generic.TemplateView):
+    form_class = PSDForm
+    initial = {'key': 'value'}
+    template_name = 'transfers/psd_dashboard.html'
+    context = {}
+
+    def get(self, request, *args, **kwargs):
+        form = self.form_class(initial=self.initial)
+        self.context = {
+            'form': form
+        }
+        return render(request, self.template_name, self.context)
```

Comment:
`get` method is not returning a `JsonResponse`, is this temporary?

Reply:
No. This will return render. The function for JsonResponse is get_PSD_data